### PR TITLE
[OSDEV-606] Contributor Sort: Allow for ascending sort of contributors on the Map page

### PR DESCRIPTION
**Pull request by Innavin369**
[OSDEV-606](https://opensupplyhub.atlassian.net/browse/OSDEV-606) Contributor Sort: Allow for ascending sort of contributors on the Map page.

The user has the following options available for sorting facilities :
* A to Z
* Z to A
* # Contributors  :arrow_down_small: 
* # Contributors :arrow_up_small:

### CODE REVIEW

**Review by roman-stolar — DISMISSED** [review]
LGTM

File: src/django/api/views/facility/facilities_view_set.py [hunk 1]
```diff
@@ -224,10 +224,17 @@ def list(self, request):
...
+
+        if (sort_by is None) or (sort_by == 'contributors_desc'):
+            order_list = ['-contributors_count', 'name']
+        elif (sort_by == 'name_asc'):
             order_list = ['name']
-        else:
+        elif (sort_by == 'name_desc'):
+            order_list = ['-name']
+        elif (sort_by == 'contributors_desc'):
             order_list = ['-contributors_count', 'name']
```
Comment (mazursasha1990) [inline src/django/api/views/facility/facilities_view_set.py:235]: Here you use check for sort_by == 'contributors_desc' the second time. Firstly it checks at line 228.

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]

File: src/django/api/views/facility/facilities_view_set.py (hunk 1 above)
Comment (roman-stolar) [reply src/django/api/views/facility/facilities_view_set.py:235]: +1 @Innavin369

Comment (Innavin369) [reply src/django/api/views/facility/facilities_view_set.py]: removed

File: doc/release/RELEASE-NOTES.md [hunk 2]
```diff
@@ -38,6 +38,7 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-893](https://opensupplyhub.atlassian.net/browse/OSDEV-893) - Introduce a temporary API endpoint switcher for list uploads to enable switching to the old list upload API endpoint if the new endpoint affects production uptime.
 
 ### Code/API changes
+* [OSDEV-606](https://opensupplyhub.atlassian.net/browse/OSDEV-606) Contributor Sort: Allow for ascending sort of contributors on the Map page. The sort_by parameter submits type of sorting order for facilities. Default sorting will be primary by public contributors count descending and secondary by name ascending/descending and contributors count ascending.
```
Comment (mazursasha1990) [inline doc/release/RELEASE-NOTES.md:41]: Please, move the description of current changes to Release 1.11.0

**Review by mazursasha1990 — CHANGES_REQUESTED** [review]

**Review by mazursasha1990 — APPROVED** [review]
LGTM

**Review by mazursasha1990 — APPROVED** [review]

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**